packages/health-check/src/metadata.js

```diff
@@ -0,0 +1,1043 @@
+const developMomentumMetadata = {
```

Comment:
Can you convert this file to multiple `.json` files and put them in `fixtures` directory?